src/Parser/Field/FieldFactory.php

```diff
@@ -102,7 +102,7 @@ public static function build(
             }
 
             //Must NOT be a repeater
-            if (! $repeatingFieldCollection) {
+            if (!isset($repeatingFieldCollection) || !$repeatingFieldCollection) {
                 $repeatingFieldCollection = null;
```

Comment:
Pro-tip, define `$repeatingFieldCollection = null;` at the top of the method then you don't need to check if it is set … and that method makes kittens cry … See, proof: :crying_cat_face: :wink: